### ISSUE TITLE
yt.title is printing result as YouTube

### ISSUE THREAD

**Issue by ValkyreQueen** [pytube/pytube#668]
dont know why yt.title is giving output as YouTube everytime for me any help plz

**Comment by sadikkuzu** [comment]
See #632

**Comment by shahraizali** [comment]
try this if that doesn't work out.

put this on `__main__.py` line 114 - 117

```
    # Fix for KeyError: 'title' issue #434
    if "title" not in self.player_config_args:  # type: ignore
        title = json.loads(self.player_config_args['player_response'])['videoDetails']['title']
        self.player_config_args["title"] = unescape(title)
```

<img width="867" alt="Screen Shot 2020-07-01 at 7 58 45 PM" src="https://user-images.githubusercontent.com/15216270/86259340-5400fb00-bbd5-11ea-89b7-8cfe308b19bc.png">

This fixed it for me

**Comment by gorkemt** [comment]
video = YouTube(url)
title = video.player_response['videoDetails']['title']
worked for me.

**Comment by ValkyreQueen** [comment]
thanks a lot sir @gorkemt it helped me

**Comment by sadikkuzu** [comment]
i've just improvised something like that:
(in `__main__.py`)
```python
    def descramble(self) -> None:
        """Descramble the stream data and build Stream instances.

        The initialization process takes advantage of Python's
        "call-by-reference evaluation," which allows dictionary transforms to
        be applied in-place, instead of holding references to mutations at each
        interstitial step.

        :rtype: None

        """
        logger.info("init started")

        self.vid_info = dict(parse_qsl(self.vid_info_raw))
        if self.age_restricted:
            self.player_config_args = self.vid_info
        else:
            assert self.watch_html is not None
            self.player_config_args = get_ytplayer_config(self.watch_html)["args"]

        # https://github.com/nficano/pytube/issues/165
        stream_maps = ["url_encoded_fmt_stream_map"]
        if "adaptive_fmts" in self.player_config_args:
            stream_maps.append("adaptive_fmts")

        # unscramble the progressive and adaptive stream manifests.
        for fmt in stream_maps:
            if not self.age_restricted and fmt in self.vid_info:
                apply_descrambler(self.vid_info, fmt)
            apply_descrambler(self.player_config_args, fmt)

            if not self.js:
                if not self.embed_html:
                    self.embed_html = request.get(url=self.embed_url)
                self.js_url = extract.js_url(self.embed_html)
                self.js = request.get(self.js_url)

            apply_signature(self.player_config_args, fmt, self.js)

            # build instances of :class:`Stream <Stream>`
            self.initialize_stream_objects(fmt)

        # load the player_response object (contains subtitle information)
        self.player_response = json.loads(self.player_config_args["player_response"])
        del self.player_config_args["player_response"]
        self.player_config_args["title"] = unescape(self.player_response['videoDetails']['title'])
        self.stream_monostate.title = self.title
        self.stream_monostate.duration = self.length

        logger.info("init finished successfully")
```

**self.player_config_args["title"] = unescape(self.player_response['videoDetails']['title'])**

**Comment by golgor** [comment]
> video = YouTube(url)
> title = video.player_response['videoDetails']['title']
> worked for me.

Great one, thanks! It works perfectly for me as well.

**Comment by cstuart1310** [comment]
> 
> 
> try this if that doesn't work out.
> 
> put this on `__main__.py` line 114 - 117
> 
> ```
>     # Fix for KeyError: 'title' issue #434
>     if "title" not in self.player_config_args:  # type: ignore
>         title = json.loads(self.player_config_args['player_response'])['videoDetails']['title']
>         self.player_config_args["title"] = unescape(title)
> ```
> 
> <img alt="Screen Shot 2020-07-01 at 7 58 45 PM" width="867" src="https://user-images.githubusercontent.com/15216270/86259340-5400fb00-bbd5-11ea-89b7-8cfe308b19bc.png">
> 
> This fixed it for me

Can confirm this works better (for me anyway) than the solution in #632
